README.md

```diff
@@ -125,6 +127,12 @@ Edit it by adding the following lines:
     (boolean)(Optional)<br/>
     If the SmartThings API is enabled (by settings "api_key" and "device_id"), then the TV Channel Names will show as media titles, by setting this to True the TV Channel Number will also be attached to the end of the media title. (when applicable)
 
+    **scan_app_ws:**<br/>
```

Comment:
There are a couple of reasons why I think this is not needed:
- it's not named properly, to get the running app (in the old way) we use HTTP polling, not WS polling, so `scan_app_ws` makes no sense
- what I said in the other comment, if all apps in `app_list` are set with `org.`, then HTTP polling is not used at all, so this is only used if the `app_list` is not set properly
- this shouldn't be set to `false` if SmartThings API is enabled, because that will just break getting the running app for all users that use this component right now, that will create issues confused as to why it's not working anymore

I would drop this completely, or just name it `show_running_app` that is always `true` and if disabled would stop the HTTP polling method. (the default value of this should never change, regardless of other options that are set)

Reply:
If I'm not wrong, if you are seeing TV using HDMI or TV input, the call to "_get_running_app" cause running both SmartThings and HTTP method, this is the reason to put this parameters. At the begining of the module during "setup_platform", if SmartThings is not enabled this parameter is forced to True. I agree that you can set this at True by default, but my goal was to limit HTTP poll al mutch as possible. For the name maybe should be changed to "CONF_SCAN_APP_HTTP"

Reply:
I changed param name to "scan_app_http"

Reply:
I still think we shouldn't turn it on by default if SmartThings is enabled. Default values should not change based on circumstance. This should be `True` by default, if users want to disable it, they can do so manually.

Reply:
This is a philosophical discussion, anyaway you are the boss :).
I set default to True and removed piece of code that force the value. Also changed little bit explanation in readme.